flocker/control/test/test_schemas.py

```diff
@@ -110,3 +110,22 @@
          u"dataset_id": u"x" * 36},
     ]
 )
+
+DatasetsArrayTests = build_schema_test(
+    name="DatasetsArrayTests",
+    schema={'$ref': '/v1/endpoints.json#/definitions/datasets_array'},
+    schema_store=SCHEMAS,
+    failing_instances=[
+        # Incorrect type
+        {},
+        # Wrong item type
+        ["string"],
+        # Failing dataset type
```

Comment:
This is subtle, explain that maximum_size has a minimum value...or use something more obviously invalid...like an invalid ipv4 address.